support/cas-server-support-actions/src/main/java/org/apereo/cas/web/flow/InitialFlowSetupAction.java

```diff
@@ -37,12 +37,35 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(InitialFlowSetupAction.class);
 
-    private final CasConfigurationProperties casProperties;
-    private final ServicesManager servicesManager;
-    private final AuthenticationServiceSelectionPlan authenticationRequestServiceSelectionStrategies;
-    private final CookieRetrievingCookieGenerator warnCookieGenerator;
-    private final CookieRetrievingCookieGenerator ticketGrantingTicketCookieGenerator;
-    private final List<ArgumentExtractor> argumentExtractors;
+    /**
+     * Protected casProperties.
+     */ 
+    protected CasConfigurationProperties casProperties;
```

Comment:
Remove all protected instance. Remove their javadocs. Only mark those as protected that are needed by your changes.

Reply:
For what I see, they all need to be protected because those variables will be accessed by configureWebflowContextForService and the constructor

Reply:
Isn't `configureWebflowContextForService` owned [by the same class](https://github.com/cifren/cas/blob/875b4754e3770a8ba3eaf7e11f5e7d03e0f54d5e/support/cas-server-support-actions/src/main/java/org/apereo/cas/web/flow/InitialFlowSetupAction.java#L88) that houses all the properties? If so, you need a field to be marked as protected only to then access from a method inside the same class that has the field.

Reply:
you are right